### DISCUSSION TITLE
Can I cancel a pending invite to collaborate in a private repository without informing the invitee?

### DISCUSSION THREAD

**Discussion by billtubbs** [community/community#21825] [New to GitHub]
<p>I invited someone to collaborate on a repository a while ago and they haven’t accepted the invite (it says ‘Pending Invite’ in settings -&gt; Manage access).</p>
<p>If I click the trash can it will presumably delete the invite but will the person I invited get a notification that the invite was cancelled?</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey! If the invite is pending and you cancel it, the recipient will not be notified. </p>
